src/shadowfinder/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .shadowfinder import ShadowFinder
+from .cli import ShadowFinderCli
+
+__all__ = ["ShadowFinder", "ShadowFinderCli"]
```

Comment:
This is something I learned recently at work, actually. `__all__` serves a few purposes:

* Explicitly states what we intend to export/make public from our package
* Helps users who might use `*` imports (like `from shadowfinder import *`) from importing things they don't need, like utility functions
* Pleases static type checkers like `mypy`, which we can add in the future :)